npm/package.json

```diff
@@ -11,7 +11,7 @@
   "main": "index.js",
   "types": "electron.d.ts",
   "dependencies": {
-    "@types/node": "^8.0.24",
+    "@types/node": "^10.1.4",
```

Comment:
This version might seem weird, but `@types/node` literally jumped skipped the `10.2.x` range:

<img width="466" src="https://user-images.githubusercontent.com/359239/57491656-066c5500-7294-11e9-9221-a5b2aacb5f4f.png">